docs/yaml-config.rst

```diff
@@ -26,7 +42,7 @@ Choose ``none`` to build none of the formats.
 
     # Don't build any extra formats
     formats:
-        - none
+        - null
```

Comment:
I think this would no work with the current implementation https://github.com/rtfd/readthedocs-build/blob/daecec8b74af1f55b0ab4c1dd53d9cf6aee20126/readthedocs_build/config/config.py#L399-L409. This should be:

```yaml
formats: null
```

Reply:
Won't this change affect current users using it in this way?

Reply:
Now we have one :) https://github.com/rtfd/readthedocs-build/pull/43

> Won't this change affect current users using it in this way?

The current description doesn't work for any user using it

Reply:
Mmm, now I see, this value is hardcoded as a string on the code. What should be done now?

Reply:
and now that I know the syntax for an empty list, would be better that this is represented as `[]`

Reply:
👍  on using the `formats: []` syntax. This example will produce something we don't want:

```
In [1]: yaml.load("foo:\n  - null")
Out[1]: {'foo': [None]}
```